tests/ui/components/filters/filteroptionscomponent.js

```diff
@@ -116,6 +116,48 @@ describe('filter options component', () => {
     expect(wrapper.find('.js-yxt-FilterOptions-showMore')).toHaveLength(1);
   });
 
+  it('renders correct options based on the searchable input', () => {
+    const config = {
+      ...defaultConfig,
+      control: 'multioption',
+      searchable: true
+    };
+    const component = COMPONENT_MANAGER.create('FilterOptions', config);
+    const wrapper = mount(component);
+    expect(options).toHaveLength(6);
+    const searchInputEl = wrapper.find('.js-yxt-FilterOptions-filter');
+    expect(searchInputEl).toHaveLength(1);
+
+    // empty input
```

Comment:
can we split each of these cases into separate tests? right now, if one case fails, we won't run any cases after it. it would also make it easier to debug if it fails in the future

Reply:
I think they fit better in a single test as the initial config settings are the same only the input in the UI changed. Ack